templates/about.html

```diff
@@ -15,9 +15,18 @@ <h1 class="end_header">About</h1>
       involved in the <em>FIRST</em> Robotics Competition (FRC) with scouting data and match videos.
       Since then, the project has grown to include developers from within the <em>FIRST</em> community from
       various teams all over the world. We continually strive to make The Blue Alliance an even more valuable
-      resource for our users.</p>
-      <p>Have a suggestion? <a href="/contact">Contact us!</a></p>
-      <p>Are you a developer? Check out our <a href="https://github.com/the-blue-alliance/the-blue-alliance" target="_blank"> GitHub repository</a> and get involved!</p>
+      resource for our users.  Other additions to The Blue Alliance include match prediction and anaytics, 
+      hosting of CAD models, and The Blue Alliance Blog.</p>
+	   
+      <div class="hidden-xs">
+      <br>
+      <div class="btn-group btn-group-block-2">
+<a class="btn btn-default" href="/contact"><span class="glyphicon glyphicon-info-sign"></span>    Contact Us!</a>
```

Comment:
I don't believe there's any difference between 1 space and multiple spaces. Same below.